Projects/UOContent/Items/Addons/SpinningwheelEastAddon.cs

```diff
@@ -2,14 +2,6 @@
 
 namespace Server.Items
 {
-    public delegate void SpinCallback(ISpinningWheel sender, Mobile from, int hue);
-
-    public interface ISpinningWheel
```

Comment:
I am not sure this should have been moved. Let's move it back?

Reply:
it is moved to elvenspinningwheelEastaddon

Reply:
This was moved over to ElvenSpinningWheelEastAddon.cs